src/pages/events/Events.css

```diff
@@ -1 +1,5 @@
-/* stylelint-disable-next-line no-empty-source */
+.events-container {
```

Comment:
Use `background-color: #242424;` for events page.

Reply:
Actually keep the `background-color` of every page as `#242424` and card-background-color as `background-color: #2b2b2b`